docs/Introduction.md

```diff
@@ -0,0 +1,74 @@
+
+## Problems solved and application scenarios in Web3 by Manta
+
+Although the current distributed ledger has brought a new payment model without the participation of third-party financial institutions, most records of the current cryptocurrency transactions are publicly available due to the transparency of blockchain technology. Public transactions can be viewed by competitors or third-party organizations, which affects business interests. At an individual level, public transactions can be taken advantage of by advertisers and criminals.
+
+Although privacy-preserving cryptocurrencies such as ZCash exist, the status quo has two significant problems. High price volatility is the first problem, which affects practical use. The lack of scalability is the second problem, which limits use to only simple payments. The exchange of privacy-preserving coins and other cryptocurrencies, including other privacy-preserving coins, is still unsolved — that still relies on centralized exchanges and OTC.
+
+Manta Network addresses the above two pain points. First, **the decentralized privacy-preserving payment protocol** based on **zk-SNARK** supports Polkadot and Substrate Parachain assets to their corresponding private pairs. In this way, users can use popular assets like stable coins, renBTC, etc., while simultaneously benefiting from the anonymity of privacy coins.
+
+Second, **a decentralized privacy-preserving exchange protocol** based on **AMM** and **zk-SNARK** can realize decentralized transactions between private coins, which is similar to Uniswap and Curve, and ensure transaction addresses' **privacy**. Our product is the only platform that can meet user privacy needs, boast anti-surveillance, and ensure private tokens' stability and interoperability. Also, compared to the centralized dark pool, the decentralized version is more credible and secure, as it reduces single points of failure.
+
+If you want to learn more, read our article: [An Introduction of Manta Network](https://mantanetwork.medium.com/an-introduction-of-manta-network-cd2dddf827cb).
+
+## Manta Network's Kusama Chain Project Calamari
+
+### About Calamari
+
+Calamari, Manta Network's canary net, is the plug-and-play privacy-preservation parachain built to service the Kusama DeFi world. It combines Kusama and zkSNARKs to bring on-chain privacy to transactions and swaps. As part of its suite of products, Calamari Network offers:
+
+### MariPay
+
+Its token-agnostic private payment service. MariPay supports the private transfer of Kusama and its Parachain assets, including major crypto assets supported on Kusama. Users can transact popular purchases like stablecoins and wrapped BTC while simultaneously reaping the benefits of on-chain privacy through ZKP.
```

Comment:
```suggestion
MariPay is a token-agnostic private payment service. MariPay supports the private transfer of Kusama and its Parachain assets, including major crypto assets supported on Kusama. Users can transact popular purchases like stablecoins and wrapped BTC while simultaneously reaping the benefits of on-chain privacy through ZKP.
```

Why is this description so different than MantaPay? Aren't they the same protocol? It seems that the best way to do this would just be to say that MariPay is the same as MantaPay, it will just have newer features. And then you can list the features that MariPay currently has that MantaPay doesn't. Same for MariSwap. If this isn't the case, these docs need to make it a lot more clear what the differences are.